app.py

```diff
@@ -17,6 +17,19 @@
             </style>
             """
 st.markdown(hide_st_style, unsafe_allow_html=True)
+pg_bg= """
+<style>
```

Comment:
Need to change the styles to another file and import in app.py for code readability